### PR TITLE
Update ProcessExportPds4 to treat input of the form filename.xml as a label name

### PR DESCRIPTION
**Pull request by krlberry**
Update ProcessExportPds4 to treat input of the form filename.xml as a label name, _not_ an image name with an unusual extension. 

Fixes the problem in which 
tgocassisrdrgen to=filename.xml 
and
isis2pds pdsversion=pds4 to=filename.xml 
produce only one output file, rather than the expected two files (one for the label, one for the image.)

### CODE REVIEW

**Comment by krlberry** [comment]
[Fix tgocassisrdrgen to=filename.xml only 1 output file problem](https://trello.com/c/Y0ONWpPR/390-fix-tgocassisrdrgen-tofilenamexml-only-1-output-file-problem)

**Review by jessemapel — COMMENTED** [review]
I get a weird error when trying to build this
```
ProcessExportPds4.cpp:481:1: error: stray ‘`’ in program
 `   QString path(outputFile.originalPath());
 ^
/scratch/test_build/ISIS3/isis/make/isismake.objs:83: recipe for target 'ProcessExportPds4.o' failed
make[1]: *** [ProcessExportPds4.o] Error 1
/scratch/test_build/ISIS3/isis/make/isismake.objs:78: recipe for target 'object' failed
make: *** [object] Error 2
```

File: isis/src/base/objs/ProcessExportPds4/ProcessExportPds4.cpp [hunk 1]
```diff
@@ -468,19 +468,29 @@ namespace Isis {
...
    *        and an IMG file with the output file name.
    *
    */
   void ProcessExportPds4::WritePds4(QString outFile) {
     
     FileName outputFile(outFile);
 
-    QString path(outputFile.originalPath());
+    // Name for output label
+`   QString path(outputFile.originalPath());
```
Comment (jessemapel) [inline isis/src/base/objs/ProcessExportPds4/ProcessExportPds4.cpp]: It looks like there's an extra back tick here

Comment (krlberry) [reply isis/src/base/objs/ProcessExportPds4/ProcessExportPds4.cpp]: Thanks for catching that. I'm not sure how it slipped in in-between testing and committing.

Comment (krlberry) [reply isis/src/base/objs/ProcessExportPds4/ProcessExportPds4.cpp]: @jessemapel Okay, fixed!

**Review by jessemapel — APPROVED** [review]
Looks good, new functionality is working and old tests are still passing.

**Pull request merged**